internal/handlers/userinfo.go

```diff
@@ -62,6 +53,62 @@ func (data UserInfoData) ToJSON() map[string]any {
 	return m
 }
 
+func (data UserInfoData) profileJSON() map[string]any {
+	if data.Profile == nil {
+		return nil
+	}
+
+	m := map[string]any{}
+	claims := make(map[string]any)
+	for k, v := range data.Profile.GetClaims().AsMap() {
+		claims[k] = v
+	}
+	m["claims"] = m
+	return m
+}
+
+func (data UserInfoData) sessionJSON() map[string]any {
+	if data.Session == nil {
+		return nil
+	}
+
+	m := map[string]any{}
+	claims := make(map[string]any)
+	for k, vs := range data.Session.GetClaims() {
+		claims[k] = vs.AsSlice()
+	}
+	m["claims"] = claims
```

Comment:
are there any claims we might want to omit here? the id token `aud` claim would have pomerium's client id, though it's not a secret and might be necessary for validation purposes

Reply:
I don't know what claims are considered sensitive.

Reply:
Me neither, but I am not 100% confident that none of them are. @wasaga do you know?